### ISSUE TITLE
 Contrast ratio for 'Next' & 'Previous' buttons present in 'Date picker' dialog is '1.4:1'.  

### ISSUE THREAD

**Issue by msftedad** [Hacker0x01/react-datepicker#3765]
**Describe the bug**
 Contrast ratio for 'Next' & 'Previous' buttons present in 'Date picker' dialog is '1.4:1'. 

**To Reproduce**
1. Open the above mentioned 'URL' in 'Microsoft Edge dev browser' & login with valid credentials.
2. 'Intern support' Home page will get open.
3. Navigate to 'Candidate' link via tab key & activate it using enter.
4. 'Candidates' detail page will get open.
5. navigate to 'New candidate' link via tab key & activate it using enter.
6. A page with 'Form' fields to create a candidate will get open.
7. Navigate to the 'Next' & 'Previous' buttons present in 'Date picker' dialog of 'Birthday', 'enroll date' & graduate date' edit fields & verify the contrast ratio for the buttons with the background.
8. Observe the issue.

**Actual behavior**
Contrast ratio for 'Next' & 'Previous' buttons present in 'Date picker' dialog is '1.4:1', which is less than '3:1' (Required).

**Expected behavior**
Contrast ratio for 'Next' & 'Previous' buttons present in 'Date picker' dialog should be greater than or equal to '3:1'.

**Screenshots**
<img width="960" alt="Screenshot 2022-09-21 215512" src="https://user-images.githubusercontent.com/93735775/191559517-69651094-6af6-4a21-91e8-907983756310.png">

**Desktop (please complete the following information):**
OS: Windows 11 Version: 21H2 (OS Build 22000.856)
Browser: Microsoft Edge Dev Version 105.0.1343.42 (Official build) (64-bit)
URL: [https://internsupport.microsoft.com/](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2finternsupport.microsoft.com%2f&data=05%7c01%7cv-knarang%40microsoft.com%7cf76091e47ae04f67a01208da90094e27%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637980666901980033%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=R1BmtgD%2f2S7RZRcKY6DbVJrEnckvGhdnsVy5XuBrbTQ%3D&reserved=0)
Username: V-ID (Microsoft ID)

Tool: 'Accessibility insights for windows'

**Smartphone (please complete the following information):**
N/A

**Additional context**
Similar issue repro, as contrast ratio for Keyboard focus indicator for the options present in the expanded combo boxes is '1.2:1'.

**Comment by msftedad** [comment]
please don't close this issue as it is still repro'ing.Thanks
